### DISCUSSION TITLE
Is there a way to change the background color of the headline?

### DISCUSSION THREAD

**Discussion by gciatto** [thegeeklab/hugo-geekblog#70] [Q&A] [answered]
More generally, parameterising the color may be useful

**Comment by xoxys** [comment]
Right now, you can add a custom.css to your project's dir and overwrite the colors using the CSS classes. For the future its planned to use custom css properties to simplify theming as already done here https://geekdocs.de/features/theming/

**Comment by xoxys** [comment, marked as answer]
[Theming](https://hugo-geekblog.geekdocs.de/posts/features/theming/) is now implemented and should simplify customizations.
